specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-10-01/updateRuns.json

```diff
@@ -0,0 +1,309 @@
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/UpdateNameParameter"
+          },
+          {
+            "$ref": "#/parameters/UpdateRunNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/UpdateRun"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Update runs under cluster resource": {
+            "$ref": "./examples/GetUpdateRuns.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "UpdateRunList": {
+      "description": "List of Update runs",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of Update runs",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/UpdateRun"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "UpdateRun": {
+      "description": "Details of an Update run",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "The geo-location where the resource lives"
+        },
+        "properties": {
+          "description": "Describes Machine Extension Properties.",
+          "$ref": "#/definitions/UpdateRunProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "UpdateRunProperties": {
+      "description": "Properties of an Update run",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state of the UpdateRuns proxy resource.",
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Accepted",
+            "Provisioning"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "timeStarted": {
```

Comment:
>timeStarted [](http://example.com/codeflow?start=9&length=11)

This and lastUpdatedTime look like they belongs in systemData of the resource. The systemData property is already part of the ProxyResource definition in v3 common types. Ref: [https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html](https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html).